### ISSUE TITLE
map enhancements

### ISSUE THREAD

**Issue by miguelcleon** [ODM2/CZ-Manager#140]
@lsetiawan @emiliom

I have two improvements to maps I'd like to make. 
1) change the way time series are listed so that related time series results are viewed as a continuous time series instead of two separate listings. I think this should be based on two results having a related results record of type continues. 
2) We already can filter the map by datasets. I'd like to add URL variables so links can be shared with the map already filtered.

**Comment by lsetiawan** [comment]
@miguelcleon I think I should finish fixing things first for the map before further changes or the code will become confusing, since I made a lot of changes for CZIMEA stuff.

**Comment by miguelcleon** [comment]
@lsetiawan Yes, agreed, I wanted to write these ideas down while we worked on completing the CZIMEA map changes.
